tests/auto/api.json

```diff
@@ -2350,6 +2429,36 @@
                 "success": "Bool",
                 "transactionId": "Int"
             }
+        },
+        "Zigbee.AdapterAdded": {
+            "description": "Emitted whenever a new Zigbee adapter candidate has been detected in the system.",
```

Comment:
I think "candidate" is not needed here

Reply:
Also, this doesn't only list Zigbee adapters but also raw serial ports... Perhaps that could be made clearer?

In general I think it might be better to refer to "Zigbee adapters" and "serial ports" instead of "recognized adapters" and "unrecognized adapters"